iocore/net/SSLNetVConnection.cc

```diff
@@ -1238,12 +1238,12 @@ SSLNetVConnection::sslServerHandShakeEvent(int &err)
     if (SSL_get_all_async_fds(ssl, &waitfd, &numfds) && numfds > 0) {
```

Comment:
Reference: https://www.openssl.org/docs/man1.1.0/man3/SSL_waiting_for_async.html
`The number of file descriptors returned is stored in *numfds and the file descriptors themselves are in *fds. The fds parameter may be NULL in which case no file descriptors are returned but *numfds is still populated. It is the callers responsibility to ensure sufficient memory is allocated at *fds so typically this function is called twice (once with a NULL fds parameter and once without).`

Memory leaks here.

Reply:
Actually no leak, but I'm playing fast and loose with the assumption that numfds is 1.  My stack allocation is for only 1 item.  Updating to make the call with nullptr for the fds to get the actual count and stack allocate the correct number of items.